im.fluffychat.Fluffychat.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "im.fluffychat.Fluffychat",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "command": "fluffychat",
+    "copy-icon": true,
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=chat.fluffy.fluffychat"
+    ],
+    "modules": [
+        {
+            "name": "olm",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.matrix.org/matrix-org/olm.git",
+                    "tag": "3.2.1"
+                }
+            ]
+        },
+        {
+            "name": "fluffychat",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -r linux/release/bundle /app/fluffychat",
+                "chmod +x /app/fluffychat/fluffychat",
+                "install -Dm644 linux/release/bundle/data/flutter_assets/assets/logo.svg /app/share/icons/hicolor/scalable/apps/im.fluffychat.Fluffychat.svg",
+                "mkdir /app/bin",
+                "ln -s /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 /app/lib/libsqlite3.so",
+                "ln -s /app/fluffychat/fluffychat /app/bin/fluffychat",
+                "install -Dm644 im.fluffychat.Fluffychat.desktop /app/share/applications/im.fluffychat.Fluffychat.desktop",
+                "install -Dm644 im.fluffychat.Fluffychat.appdata.xml /app/share/appdata/im.fluffychat.Fluffychat.appdata.xml"
+
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/ChristianPauly/fluffychat-flutter/-/jobs/809187142/artifacts/download?file_type=archive",
+                    "dest-filename": "artifacts.zip",
+            
+                    "sha256": "ef075736781dad6828f8abac02955966e53d37b6508caf7af6b85f44155aba4b"
+                },
+                {
+                    "type": "file",
+                    "path": "im.fluffychat.Fluffychat.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html